doc/nrf/known_issues.rst

```diff
@@ -679,45 +679,45 @@ Closing sockets
 Multiprotocol Service Layer (MPSL)
 ==================================
 
-.. rst-class:: v1-4-99-dev1 v1-4-2 v1-4-1
+.. rst-class:: v1-5-0 v1-4-2 v1-4-1
 
 DRGN-15223: `CONFIG_SYSTEM_CLOCK_NO_WAIT` is not supported for nRF5340
   Using :option:`CONFIG_SYSTEM_CLOCK_NO_WAIT` with nRF5340 devices might not work as expected.
 
-.. rst-class:: v1-4-2 v1-4-1
+.. rst-class:: v1-5-0 v1-4-2 v1-4-1
 
 DRGN-15176: `CONFIG_SYSTEM_CLOCK_NO_WAIT` is ignored when Low Frequency Clock is started before initializing MPSL
```

Comment:
@rugeGerritsen or @hurricaneFromMoscow  This one is fixed, right?

Reply:
@rugeGerritsen , @hurricaneFromMoscow , any updates on this?

Reply:
It is fixed in 1.5, yes